pom.xml

```diff
@@ -371,8 +366,8 @@
     </dependencyManagement>
 
     <modules>
-        <module>deploy</module>
-        <module>docs</module>
+        <!--<module>deploy</module>-->
+        <!--<module>docs</module>-->
```

Comment:
Why are you commenting these out?

Reply:
And is the surprise addition of the license headers somehow related to these being commented out?

Reply:
Updated